test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlCommentsDocumentFilterTests.cs

```diff
@@ -51,5 +53,114 @@ private static XmlCommentsDocumentFilter Subject()
+                        }
+                    },
+                    new ApiDescription
+                    {
+                        ActionDescriptor = new ControllerActionDescriptor
+                        {
+                            ControllerTypeInfo = typeof(FakeControllerWithXmlComments).GetTypeInfo(),
+                            ControllerName = nameof(FakeControllerWithXmlComments),
+                            RouteValues = new Dictionary<string, string> { { "controller", expectedTagName } }
+                        }
+                    }
+                },
+                null,
+                null);
+
+            Subject(options).Apply(document, filterContext);
+
+            var tag = Assert.Single(document.Tags);
+            Assert.Equal(expectedTagName, tag.Name);
+        }
+
+        [Fact]
+        public void Uses_Proper_Tag_Name_With_Custom_TagSelector()
+        {
+            var expectedTagName = "AliasControllerWithXmlComments";
+            var options = new SwaggerGeneratorOptions { TagsSelector = apiDesc => new[] { expectedTagName } };
+            var document = new OpenApiDocument();
+            var filterContext = new DocumentFilterContext(
+                new[]
+                {
+                    new ApiDescription
+                    {
+                        ActionDescriptor = new ControllerActionDescriptor
+                        {
+                            ControllerTypeInfo = typeof(FakeControllerWithXmlComments).GetTypeInfo(),
+                            ControllerName = nameof(FakeControllerWithXmlComments),
+                        }
+                    },
+                    new ApiDescription
+                    {
+                        ActionDescriptor = new ControllerActionDescriptor
+                        {
+                            ControllerTypeInfo = typeof(FakeControllerWithXmlComments).GetTypeInfo(),
+                            ControllerName = nameof(FakeControllerWithXmlComments),
+                        }
+                    }
+                },
+                null,
+                null);
+
+            Subject(options).Apply(document, filterContext);
+
+            var tag = Assert.Single(document.Tags);
+            Assert.Equal(expectedTagName, tag.Name);
+        }
+
+        private static XmlCommentsDocumentFilter Subject(SwaggerGeneratorOptions options)
+        {
+            using (var xmlComments = File.OpenText($"{typeof(FakeControllerWithXmlComments).Assembly.GetName().Name}.xml"))
+            {
+                return new XmlCommentsDocumentFilter(new XPathDocument(xmlComments), options);
+            }
+        }
+
+        [Fact]
+        public void Ensure_IncludeXmlComments_Adds_Filter_To_Options()
+        {
+            var services = new ServiceCollection();
+            services.AddSingleton<Microsoft.AspNetCore.Hosting.IWebHostEnvironment, DummyHostEnvironment>();
```

Comment:
As it's used in two places, let's add the `using` at the top of the file.

```suggestion
            services.AddSingleton<IWebHostEnvironment, DummyHostEnvironment>();
```

Reply:
done